### ISSUE TITLE
input size 416 -> 832 makes nano  not converging on COCO and custom dataset

### ISSUE THREAD

**Issue by liuruijin17** [Megvii-BaseDetection/YOLOX#716]
Hi, thanks for your excellent work!

input size = 416 works well on the custom dataset, but when changing to 832, the nano did not converge in 300 epochs (AP is always 0, loss keeps 10 - 11)

Here is the exp file
[nano.py.zip](https://github.com/Megvii-BaseDetection/YOLOX/files/7189483/nano.py.zip)

BTW, I try on COCO when changing input size from 416 -> 832, the nano also did not converge (loss keeps 5 - 6). This might be a bug.

Both use the pre-trained model you provided.

**Comment by Joker316701882** [comment]
@liuruijin17 Did you use cache? When you change the image size, you need to delete the existing cache and re-generate it.

**Comment by liuruijin17** [comment]
Ops, it's my mistake, I delete and re-generate it, then everything goes fine. Thanks a lot!
